ext/pdo_mysql/tests/bug66528.phpt

```diff
@@ -44,7 +43,8 @@ try {
 --CLEAN--
 <?php
 require __DIR__ . '/mysql_pdo_test.inc';
-MySQLPDOTest::dropTestTable();
+$db = MySQLPDOTest::factory();
+$db->exec('DROP TABLE IF EXISTS test_66528');
```

Comment:
It might make sense to keep this method and just pass a string to it with the table nape